clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -8160,6 +8160,43 @@ void StreamAPICallRule::runRule(const MatchFinder::MatchResult &Result) {
     EA.applyAllSubExprRepl();
     return;
   }
+  if (FuncName == "cudaStreamIsCapturing") {
+    if (!DpctGlobalInfo::useExtGraph()) {
+      report(CE->getBeginLoc(), Diagnostics::TRY_EXPERIMENTAL_FEATURE, false,
```

Comment:
Pls add lit test for this warning msg.

Reply:
LIT test for warning is already here: https://github.com/oneapi-src/SYCLomatic/blob/351f11b45deb685500d149a7130c62f21845381a/clang/test/dpct/cudaGraph_test_default_option.cu#L56